libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -230,9 +231,7 @@ static int location_project(lua_State *L) {
         return luaL_argerror(L, args, "too many arguments");
     }
 
-    location_update(*check_location(L, -3),
-                    luaL_checknumber(L, -2),
-                    luaL_checknumber(L, -1));
+    *check_location(L, -3).offset_bearing(luaL_checknumber(L, -2), luaL_checknumber(L, -1));
```

Comment:
Have you made sure scripting still compiles after this?

Something about `--scripting` to `waf`


We've broken this before...